tools/pontoon/translate.md

```diff
@@ -40,11 +40,13 @@ To switch to Review and Direct Translation Mode, go to the settings menu by clic
 
 ### Review a suggestion
 
-To review a suggestion, search for suggested strings or click on a suggested string in the sidebar.
+To review a suggestion, search for suggested strings, unreviewed suggestions or click on a suggested string in the sidebar.
```

Comment:
You could also leave suggested strings out. Or you'll need to do that when we remove them as part of [bug 1377969](https://bugzilla.mozilla.org/show_bug.cgi?id=1377969).